src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/UsageBasedMetadataManager.cs

```diff
@@ -39,7 +39,7 @@ public sealed class UsageBasedMetadataManager : GeneratingMetadataManager
 
         internal readonly UsageBasedMetadataGenerationOptions _generationOptions;
 
-        private readonly FeatureSwitchHashtable _featureSwitchHashtable;
+        private readonly LinkAttributesHashTable _featureSwitchHashtable;
```

Comment:
Let's get rid of all of the bad copypasting

```suggestion
        private readonly LinkAttributesHashTable _linkAttributesHashTable;
```